src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
-      }
-    },
-    "picture": "lisa-hong.webp"
+    "picture": "lisa-hong.webp",
+    "positions": { "S21": { "title": "Create Officer", "tier": 16 } }
   },
   {
     "fullName": "Joshua Hughes",
-    "positions": {
-      "S21": {
-        "title": "ICC Representative",
-        "tier": 8
-      }
-    },
-    "picture": "joshua-hughes.webp"
+    "picture": "joshua-hughes.webp",
+    "positions": { "S21": { "title": "ICC Representative", "tier": 23 } }
   },
   {
     "fullName": "Iftekharul Islam",
-    "positions": {
-      "S22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "iftekharul-islam.webp"
+    "picture": "iftekharul-islam.webp",
+    "positions": { "S22": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Ibrahim Israr",
-    "positions": {
-      "S22": {
-        "title": "Secretary",
-        "tier": 3
-      }
-    },
-    "picture": "ibrahim-israr.webp"
+    "picture": "ibrahim-israr.webp",
+    "positions": { "S22": { "title": "Secretary", "tier": 3 } }
   },
   {
     "fullName": "Joel Anil John",
+    "picture": "joel-anil-john.webp",
     "positions": {
-      "S22": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "F22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "joel-anil-john.webp"
+      "S22": { "title": "Dev Officer", "tier": 19 },
+      "F22": { "title": "Algo Officer", "tier": 12 }
+    }
   },
   {
     "fullName": "Andy Lasso",
-    "positions": {
-      "F21": {
-        "title": "Dev Officer",
-        "tier": 21
-      }
-    },
-    "picture": "andy-lasso.webp"
+    "picture": "andy-lasso.webp",
+    "positions": { "F21": { "title": "Dev Officer", "tier": 19 } }
   },
   {
     "fullName": "Andrew Lau",
-    "positions": {
-      "S21": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "andrew-lau.webp"
+    "picture": "andrew-lau.webp",
+    "positions": { "S21": { "title": "Treasurer", "tier": 4 } }
   },
   {
     "fullName": "Minh Le",
-    "positions": {
-      "S22": {
-        "title": "Dev Officer",
-        "tier": 21
-      }
-    },
-    "picture": "minh-le.webp"
+    "picture": "minh-le.webp",
+    "positions": { "S22": { "title": "Dev Officer", "tier": 19 } }
   },
   {
     "fullName": "Nguyen Le",
-    "positions": {
-      "S22": {
-        "title": "Data Analyst",
-        "tier": 7
-      }
-    },
-    "picture": "nguyen-le.webp"
+    "picture": "nguyen-le.webp",
+    "positions": { "S22": { "title": "Data Analyst", "tier": 5 } }
   },
   {
     "fullName": "Tommy Le",
+    "picture": "tommy-le.webp",
     "positions": {
-      "F21": {
-        "title": "Treasurer",
-        "tier": 4
-      },
-      "S22": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "tommy-le.webp"
+      "F21": { "title": "Treasurer", "tier": 4 },
+      "S22": { "title": "Treasurer", "tier": 4 }
```

Comment:
The updated "tier" for Treasurer is '3' (both lines 149 and 150)

` "Treasurer": { "id": 3, "index": 250 },`